core/src/main/java/org/apache/accumulo/fate/zookeeper/ZooCache.java

```diff
@@ -63,6 +63,12 @@
 
   private volatile boolean closed = false;
 
+  public static final String[] extraConfigs = {"table.split.endrow.size.max",
```

Comment:
I agree.  I believe that I might have found only a subset of the list of things that potentially may have watches put on them but are never really created.  I will investigate this.  

This code change assumes a node exists until proven otherwise so it is safe.  Watchers will be put on the same Znodes the way they usually are if they are not in the list of "extraConfigs".  "extraConfigs" should probably be called "possiblyExistingAndWatchableConfigs" ..